lib/y2logsstep.pm

```diff
@@ -271,6 +282,28 @@ sub post_fail_hook {
     # Check VM load
     script_run('for run in {1..3}; do echo "RUN: $run"; vmstat; sleep 5; done | tee /tmp/cpu_mem_usage.log');
     upload_logs '/tmp/cpu_mem_usage.log';
+
+    $self->save_and_upload_log('pstree',  '/tmp/pstree');
+    $self->save_and_upload_log('ps auxf', '/tmp/ps_auxf');
+
+    # Collect yast2 installer trace
+    if (!script_run(qq{ps -eo pid,comm | grep -i [y]2start | cut -f 2 -d " " > /dev/$serialdev}, 0)) {
+        chomp(my $installer_pid = wait_serial(qr/^[\d{4}]/, 10));
+        my $trace_timeout = 120;
+        my $strace_ret = script_run("timeout $trace_timeout strace -f -o /installer_trace.log -tt -p $installer_pid", ($trace_timeout + 5));
```

Comment:
Can we use /tmp directory here. I'm working on this ticket when fs becomes read-only and mount /tmp to shared memory, so it may be the only writable directory. So would be better to use it.